app/assets/stylesheets/profile-preview-card.scss

```diff
@@ -8,9 +8,33 @@
 
   &__content.crayons-dropdown:hover {
     display: block;
+    animation: hoverAppear 500ms;
+
+    &.showing {
+      animation: none;
+    }
   }
 
   &__trigger:hover + .profile-preview-card__content.crayons-dropdown {
     display: block;
+    animation: hoverAppear 500ms;
+
+    &.showing {
+      animation: none;
+    }
+  }
+
+  @keyframes hoverAppear {
+    0% {
+      opacity: var(--opacity-0);
+    }
+
+    99% {
```

Comment:
Question (non-blocking): I'm not understanding the purpose of having 99% remaining as opacity 0.

Reply:
It's to stop a fade-in effect & make it look more like an 'appear with full opacity'

Reply:
Normally I'd use a `transition-delay` of 500ms and a `transition-duration` of 0, but I wasn't keen on adding the zero opacity to the dropdown by default, just for this hover use-case 🤔